src/components/UserProfile/ProfileHeader/index.tsx

```diff
@@ -0,0 +1,87 @@
+import Link from 'next/link';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { Permission, User, useUser } from '../../../hooks/useUser';
+import Button from '../../Common/Button';
+
+interface ProfileHeaderProps {
+  user: User;
+}
+
+const ProfileHeader: React.FC<ProfileHeaderProps> = ({ user }) => {
+  const intl = useIntl();
+  const { user: loggedInUser, hasPermission } = useUser();
+
+  return (
+    <div className="relative z-40 mt-6 mb-12 md:flex md:items-end md:justify-between md:space-x-5">
+      <div className="flex items-end space-x-5 justify-items-end">
+        <div className="flex-shrink-0">
+          <div className="relative">
+            <img
+              className="w-24 h-24 bg-gray-600 rounded-full"
+              src={user.avatar}
+              alt=""
+            />
+            <span
+              className="absolute inset-0 rounded-full shadow-inner"
+              aria-hidden="true"
+            ></span>
+          </div>
+        </div>
+        <div className="pt-1.5">
+          <h1 className="flex flex-col mb-1 sm:items-center sm:flex-row">
+            <Link
+              href={
+                user.id === loggedInUser?.id ? '/profile' : `/users/${user.id}`
+              }
+            >
+              <a className="text-lg font-bold text-transparent sm:text-2xl bg-clip-text bg-gradient-to-br from-indigo-400 to-purple-400 hover:to-purple-200">
+                {user.displayName}
+              </a>
+            </Link>
+            {user.email && (
+              <span className="text-sm text-gray-400 sm:text-lg sm:ml-2">
+                ({user.email})
+              </span>
+            )}
+          </h1>
+          <p className="text-sm font-medium text-gray-400">
+            Joined {intl.formatDate(user.createdAt)} |{' '}
+            {intl.formatNumber(user.requestCount)} Requests
+          </p>
+        </div>
+      </div>
```

Comment:
```suggestion
        <div className="text-white pt-1.5">
          <h1 className="flex flex-col items-end mb-1 text-2xl lg:text-4xl sm:flex-row">
            {user.displayName}
            {user.email && (
              <span className="text-2xl sm:ml-2">({user.email})</span>
            )}
          </h1>
          <p className="mt-1 text-xs lg:text-base lg:mt-0">
            Joined {intl.formatDate(user.createdAt)} |{' '}
            {intl.formatNumber(user.requestCount)} Requests
          </p>
        </div>
```

I know you like your purple gradient headings, but I feel like since this page sort of has the same header format as the movie/series detail pages, the styling should match.  Also, I think the link just leads to the page you're already on?

Reply:
Ah wait, I see that the link actually is necessary from the settings pages...

Reply:
Maybe we could add a "Profile" button next to the "Settings" button?  I think it's also weird that the "Settings" button is visible on the user settings pages.

Reply:
I will have it change to a profile button when on the settings page.